.cirrus/Dockerfile.ubuntu16.04

```diff
@@ -31,3 +31,23 @@ RUN set -e -x ;\
     cmake -DCMAKE_INSTALL_PREFIX=/usr . ;\
     make -j $(nproc) ;\
     make install
+
+RUN set -e -x ;\
+    mkdir -p /usr/local/src ;\
+    cd /usr/local/src ;\
+    git clone --recursive https://github.com/YosysHQ/yosys.git ;\
+    cd yosys ;\
+    git reset --hard 47a5dfdaa4bd7d400c6e3d58476de80904df460d ;\
+    make -j $(nproc) ;\
+    make install
```

Comment:
`rm -rf /usr/local/src/yosys`, at the end as the source/build files are pretty large and we don't need them (vs. trellis).